package/MDAnalysis/analysis/psa.py

```diff
@@ -338,7 +338,7 @@ def get_coord_axes(path):
 
 
 def hausdorff(P, Q):
-    r"""Calculate the Hausdorff distance between two paths.
+    r"""Calculate the undirected Hausdorff distance between two paths.
```

Comment:
@tylerjereddy , sorry, I have to correct myself: the paper (Huttenlocher et al) that I cited does *not* explicitly call the Hausdorff distance *H(A, B) = max(h(A,B), h(B,A))* in Eq 1 the "symmetric Hausdorff distance". It's just "the Haudorff distance".

The distance *h(A, B)* is called the "directed Hausdorff distance", just as you said.

If you have other citations that refer to the symmetric by any other name than just "Hausdorff distance" then use that, otherwise we should be consistent with the literature and just name it "Hausdorff distance". The docs can then briefly allude to the fact that this is a symmetric quantity.